test/prompConfig.js

```diff
@@ -0,0 +1,85 @@
+import test from 'ava';
+import path from 'path';
+import chalk from 'chalk';
+import json from 'json-extra';
+import bddStdin from 'bdd-stdin';
+import getConfig from '../lib/getConfig';
+import promptConfig from '../lib/promptConfig';
+
+const cwd = process.cwd();
+
+test('get configuration file equals .sgcrc_default', (t) => {
+  const cli = promptConfig;
+  const config = cli.config();
+  t.deepEqual(config, json.readToObjSync(path.join(cwd, '.sgcrc_default')));
+});
+
+test('choises are the same as choises generated from .sgcrc_default', (t) => {
+  const sgc = getConfig();
+  const cli = promptConfig;
+  const config = cli.config();
+  const choices = cli.choices(config);
+  const choicesList = [];
+
+  Promise.resolve(() => {
+    sgc.types.forEach((type) => {
+      const emoji = `${type.emoji} ` || '';
+      const configType = type.type;
+      const description = type.description || '';
+
+      choicesList.push({
+        value: emoji + configType,
+        name: `${chalk.bold(configType)} ${description}`,
+      });
+    }).then(() => {
+      t.deepEqual(choices, choicesList);
+    });
+  });
+});
+
+test('check the values of the question object', (t) => {
+  const cli = promptConfig;
+  const config = cli.config();
+  const choices = cli.choices(config);
+  const questions = cli.questions(choices);
+
+  t.deepEqual(questions[0].type, 'list');
+  t.deepEqual(questions[0].name, 'type');
+  t.deepEqual(questions[0].message, 'Select the type of your commit:');
```

Comment:
I think non of the questions should be tested. The only thing what you can test is:

- Is the return of `questions` an array
- is there at least one object inside